dts/arm/st/stm32l4.dtsi

```diff
@@ -90,6 +90,15 @@
 			label = "UART_3";
 		};
 
+		lpuart1: serial@40008000 {
+			compatible = "st,stm32-lpuart";
+			reg = <0x40008000 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB1_2 0x00004000>;
```

Comment:
It should be `clocks = <&rcc STM32_CLOCK_BUS_APB1_2 0x00000001>;` as bit 0 is used on L4.

Reply:
Thanks, I'll look at that!

Reply:
Yes, it's just stupid copy-paste mistakes. I'll fix that.